producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtInternalEndpointPortTypeTest.java

```diff
@@ -61,53 +67,50 @@ public class SdtInternalEndpointPortTypeTest extends AbstractSdtUnitTestBase {
     /**
      * Mocked IWsUpdateItemHandler instance.
      */
+    @Mock
     private IWsUpdateItemHandler mockUpdateItemHandler;
 
     /**
      * Set up common for all tests.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        portType = new SdtInternalEndpointPortType();
+        MockitoAnnotations.openMocks(this);
 
-        mockUpdateItemHandler = EasyMock.createMock(IWsUpdateItemHandler.class);
+        portType = new SdtInternalEndpointPortType();
         portType.setUpdateItemHandler(mockUpdateItemHandler);
-
     }
 
     /**
      * Test update item method completes successfully.
      */
     @Test
-    public void testUpdateItemSuccess() {
-        EasyMock.expect(mockUpdateItemHandler.updateItem(EasyMock.anyObject(UpdateRequestType.class))).andReturn(
-                createUpdateResponse());
-        EasyMock.replay(mockUpdateItemHandler);
+    void testUpdateItemSuccess() {
+        when(mockUpdateItemHandler.updateItem(any())).thenReturn(createUpdateResponse());
 
         final UpdateResponseType response = portType.updateItem(createUpdateRequest());
-        EasyMock.verify(mockUpdateItemHandler);
-        assertNotNull("Response expected", response);
+        verify(mockUpdateItemHandler).updateItem(any());
+        assertNotNull(response, "Response expected");
     }
 
     /**
      * Test update item method handles exceptions successfully.
      */
     @Test
-    public void testUpdateItemException() {
-        EasyMock.expect(mockUpdateItemHandler.updateItem(EasyMock.anyObject(UpdateRequestType.class))).andThrow(
-                new RuntimeException("test"));
-        EasyMock.replay(mockUpdateItemHandler);
+    void testUpdateItemException() {
+        when(mockUpdateItemHandler.updateItem(any())).thenThrow(new RuntimeException("test"));
 
         try {
             portType.updateItem(createUpdateRequest());
             fail("Runtime exception should have been thrown");
         } catch (final RuntimeException re) {
-            assertEquals("",
+            assertEquals(
                     "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage());
+                    re.getMessage(), "");

```

Comment:
Might as well use the version of assertEquals() that doesn't have the message parameter.